R/util.R

```diff
@@ -1375,7 +1375,7 @@ do_on_each_group <- function(df, func, params = quote(list()), name = "tmp", wit
   name <- avoid_conflict(colnames(df), name)
   # This is a list of arguments in do clause
   args <- append(list(quote(.)), rlang::call_args(params))
-  call <- rlang::new_call(func, rlang::as_pairlist(args))
+  call <- rlang::new_call(func, as.pairlist(args))
```

Comment:
Add either `base::`, or if it is too much, add a comment that we explicitly use `base::as.pairlist` to avoid using `rlang::as_pairlist`.